lib/watcher.js

```diff
@@ -18,6 +19,7 @@ function rethrowAsync(err) {
 
 const MIN_DEBOUNCE_DELAY = 10;
 const INITIAL_DEBOUNCE_DELAY = 100;
+const rerunMessage = chalk.gray.dim(`To rerun all tests, type 'r', followed by Enter\nTo update snapshots used in the previous tests, type 'u', followed by Enter\n`);
```

Comment:
Looking at this in my terminal I think this should just be `chalk.gray`. But we then should also update the timestamp output in the mini & verbose loggers to not be dimmed.